app/script/setup-flutter.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+if [ "$GAE_SERVICE" != "analyzer" ];
+then
+  echo "Flutter setup aborted: GAE_SERVICE must be \"analyzer\"."
+  exit 0
+fi
+
+if [[ -z "$FLUTTER_SDK" ]];
+then
+  echo "Flutter setup aborted: FLUTTER_SDK must point to a target directory."
+  exit 0
+fi
+
+if [ -d "$FLUTTER_SDK" ];
+then
+  echo "Flutter setup aborted: $FLUTTER_SDK already exists, assuming proper setup."
```

Comment:
It can exists if you run the script from local dev environment (which I do - without running docker).